README.md

```diff
@@ -28,27 +36,34 @@ follow [these instructions](#2-configuration) for install and configuration.
   - New registration - `Mattermost MS Calendar Plugin`
 - `Certificates & secrets`
   - New client secret
-- `API permissions` -> `MsGraph` -> `calendars`
-  - add needed permissions
-  - (Read, Read.Shared, ReadWrite, ReadWrite.Shared)
 - `Authentication`
   - Redirect URI -> `<MM_SITEURL>/plugins/com.mattermost.mscalendar/oauth2/complete`
     - For development (use ngrok.io URL)
+- `API permissions` -> `Microsoft Graph`
 
-### Step 2 Configure Plugin Settings
+  - Delegated permissions:
+    - Calendars.ReadWrite
+    - Calendars.ReadWrite.Shared
+    - MailboxSettings.Read
 
-**`System Console` > `PLUGINS` > `Microsoft Calendar`**
+  - Application permissions:
+    - Calendars.Read
+    - MailboxSettings.Read
+    - User.ReadAll
 
-Personal
+### Step 2: Configure Plugin Settings
 
-- `Admin User IDs` - Add your sysadmin user ID
-- `tenantID` - Leave as “common"
+**`System Console` > `PLUGINS` > `Microsoft Calendar`**
+
+- `Admin User IDs` - List of user IDs to manage the plugin
+- `tenantID` - copy from Azure App
 - `clientID` - copy from Azure App
 - `Client Secret` - copy from Azure App
 
-Enterprise
+### Step 3: Configure Bot Account
 
-- `Admin User IDs` - Add your sysadmin user ID
-- `tenantID` - copy form Azure App
-- `clientID` - copy from Azure App
-- `Client Secret` - copy from Azure App
+- Create a dedicated user in Azure to be linked to the bot
+- Log in as the dedicated user in your browser
```

Comment:
@larkox The instruction here is to log into Azure with the dedicated user. The admin will never log in as the bot in Mattermost. As far as how the user is logged into Mattermost, desktop and mobile should work, though I'd be surprised if an admin wants to go through installation steps on their phone.

The reason I have the steps in this order, is because they can accidentally connect the bot account to their personal Azure account, if they are currently logged into Azure with their personal account.